### PR TITLE
added username and group to logrotate config and changed from using s…

### PR DESCRIPTION
**Pull request by kabamwchow**
…ymlink to just copying the logrotate config over, because there may be issues with using a symlink

### CODE REVIEW

File: scripts/setup.sh [hunk 1]
```diff
@@ -5,14 +5,11 @@ sudo systemctl stop unityCacheServer
...
 sudo touch /var/log/unity/unityCacheServer.log
-#sudo groupadd unitycache
-#sudo useradd -MG unity unity
-#sudo usermod -L unity
 sudo chown unity /var/log/unity
 sudo chgrp unity /var/log/unity
 sudo chown unity /var/log/unity/unityCacheServer.log
 sudo chgrp unity /var/log/unity/unityCacheServer.log
-sudo ln -s /opt/unityCacheServer/config/unityCacheServer.logrotate /etc/logrotate.d/unityCacheServer.logrotate
+sudo cp /opt/unityCacheServer/config/unityCacheServer.logrotate /etc/logrotate.d/unityCacheServer.logrotate
```
Comment (kungfubozo) [inline scripts/setup.sh]: have we tested this locally to confirm first ?

Comment (kabamwchow) [reply scripts/setup.sh]: Tested again this time with the correct environment.

Comment (kungfubozo) [reply scripts/setup.sh]: i am confused. we have 2 PR for the same file... with almost identical changes ?

Comment (kungfubozo) [reply scripts/setup.sh]: oh sorry, brain fart. clicking on the wrong link and confused myself.... ignore the 2x PR thing ;-p

**Pull request merged**